src/console/dbseed/db-seed.console.ts

```diff
@@ -76,6 +78,7 @@ export class DbSeedConsole extends CommandRunner {
                 }
                 await this.orm.em.flush();
                 this.logger.info(`Created seed data from ${subDir} successfully.`);
+                await this.ldapClient.disconnect();
```

Comment:
Could the ldapClient.disconnect() be done in the ldap-module (or in the client provider) as a onModuleDestroy-Callback? That way we wouldn't need to call this explicitly ourselves